### PR TITLE
Add homepage, twitter and G+ icons

### PR DESCRIPTION
**Pull request by tioirawan**
This adds homepage, twitter, and gplus icons and links

Closes https://github.com/coala/gci-leaders/issues/64

### CODE REVIEW

File: templates/main.html [hunk 1]
```diff
@@ -128,4 +157,4 @@ <h3>
     <script src="static/js/app.js"></script>
     <script src="static/js/twitter.js"></script>
   </body>
-</html>
+</html>
```
Comment (andrewda) [inline templates/main.html]: EOF

File: templates/main.html [hunk 2]
```diff
@@ -92,26 +107,40 @@ <h3>
...
+              </a>
+            {{/twitter_url}}
+            {{#gplus_url}}
+              <a href="{{gplus_url}}">
+                <img src="static/images/logos/gplus.png" class="chat" />
+              </a>
+            {{/gplus_url}}
+            {{#website_url}}
+              <a href="{{website_url}}">
+                <img src="https://{{image_url}}" class="chat" />
```
Comment (andrewda) [inline templates/main.html]: Don't use the org's image for their website prompt. Usually a globe is used, similar to https://image.flaticon.com/icons/png/128/12/12195.png (you might need to find one from somewhere else or give proper attribution for that one).

**Review by andrewda — CHANGES_REQUESTED** [review]
Both icons should be resized to 72px like the other icons. Crop the twitter logo first so it doesn't look weirdly small. Other than these small changes, looks good!

**Comment by jayvdb** [comment]
(also rebase please ;-)

**Comment by jayvdb** [comment]
I think we should have a strong horizontal line between each org, and a much less visually distracting separator between icons and participants.

**Comment by tioirawan** [comment]
Where to put horizontal line? if i put outside orgs div, it doesn't show, if i put it inside orgs div, it looks weird

**Comment by blazeu** [comment]
You can use css border instead of `<hr>`,

File: static/css/main.css [hunk 3]
```diff
@@ -25,7 +25,11 @@ a {
 
 .org {
   margin: 1em 0;
+  padding: 10px;
```
[the remark this answers is not in the document]
Comment (tioirawan) [reply static/css/main.css]: Solution?

[the remark this answers is not in the document]
Comment (blazeu) [reply static/css/main.css]: Try adding

```css
html {
  box-sizing: border-box;
}

*, *:before, *:after {
  box-sizing: inherit;
}
```

**Comment by blazeu** [comment]
I think we only need `border-bottom` here,
and set the color to grey-ish

File: static/css/main.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (tioirawan) [reply static/css/main.css]:
```
static/css/main.css
|   2| ··box-sizing:·border-box;
|    | [NORMAL] CSSLintBear:
|    | The box-sizing property isn't supported in IE6 and IE7. (box-sizing)
```

[the remark this answers is not in the document]
Comment (Indmind) [reply static/css/main.css]:
```
static/css/main.css
|   2| ··box-sizing:·border-box;
|    | [NORMAL] CSSLintBear:
|    | The box-sizing property isn't supported in IE6 and IE7. (box-sizing)
```

[the remark this answers is not in the document]
Comment (tioirawan) [reply static/css/main.css]:
```
static/css/main.css
|   5| *,·*:before,·*:after·{
|    | [NORMAL] CSSLintBear:
|    | The universal selector (*) is known to be slow. (universal-selector)
```

[the remark this answers is not in the document]
Comment (blazeu) [reply static/css/main.css]: ^ Crap, we kinda need it tho.
I'll add it later.

**Comment by blazeu** [comment]
>.. and a much less visually distracting separator between icons and participants.

^ add that.

**Comment by blazeu** [comment]
Also,
>icons should be resized to 72px

**Comment by blazeu** [comment]
And I think @andrewda wants you to add an attribution (like the google logo) to the icon or find another icon that's free to use without attribution.

**Comment by tioirawan** [comment]
> .. and a much less visually distracting separator between icons and participants.

I still don't understand this

**Comment by tioirawan** [comment]
> And I think @andrewda wants you to add an attribution (like the google logo) to the icon or find another icon that's free to use without attribution.

so where should I put the attribution, in the footer?

**Comment by blazeu** [comment]
>I still don't understand this

He wants you to add border below the icon, **But do not do it, for now**
I think it is hard to do anything without proper box-sizing.

Just add `margin-top: 1em;` to `.org-leaderboard` to give some spacing.

>so where should I put the attribution, in the footer?

Yeah, that'll work.

**Comment by blazeu** [comment]
Also remove this

>I think we only need border-bottom here,
>and set the color to grey-ish

**Comment by tioirawan** [comment]
I get free images from this https://www.iconfinder.com/

File: static/css/main.css [hunk 4]
```diff
@@ -25,6 +25,7 @@ a {
 
 .org {
   margin: 1em 0;
+  padding-bottom: 1em;
```
Comment (blazeu) [inline static/css/main.css]: Let's not modify this for now.

File: templates/main.html [hunk 5]
```diff
@@ -86,7 +100,6 @@ <h3>
     </div>
     {{#noLeader.length}}
       <h3>Organization(s) with no listed leaders</h3>
-      <hr>
```
Comment (blazeu) [inline templates/main.html]: Add this back

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/68/commits/44c92e30276de62027d1d773628d6200f9776304

**Review by blazeu — APPROVED** [review]

**Pull request merged**